account_banking_mandate/models/account_banking_mandate.py

```diff
@@ -145,6 +145,14 @@ def create(self, vals=None):
             vals['unique_mandate_reference'] = \
                 self.env['ir.sequence'].next_by_code(
                     'account.banking.mandate') or 'New'
+        if self._context.get('mandate_bank_partner_view'):
```

Comment:
@rgarnau You just can remove 'default_partner_bank_id' from the view, and it will work without needing all this code. Please can you check it out?

Reply:
@oihane - Thanks for your help! It works but then I cannot validate the mandate right away as the information passed it's not ready until you save (if you validate it gives you an error that bank account is missing). In order to fix this, I'm adding the option of validating in the tree view and hiding it from the mandate when creating from a res.partner.bank. What do you think?